images/operator/Dockerfile

```diff
@@ -69,3 +70,26 @@ COPY --from=builder /out/${TARGETOS}/${TARGETARCH}/LICENSE.all /LICENSE.all
 WORKDIR /
 ENV GOPS_CONFIG_DIR=/
 CMD ["/usr/bin/cilium-${OPERATOR_VARIANT}"]
+
+FROM --platform=${BUILDPLATFORM} ${CILIUM_BUILDER_IMAGE} as debug-tools
+
+#
+# Cilium Operator debug image.
+#
+# Typical image bulids will stop above at the 'release' target, but
+# developers follow this Dockerfile to the end. Starting from a release
+# image, install delve debugger and wrap the cilium-operator binary calls
+# with a script that automatically provisions the debugger on a
```

Comment:
Super nit:

```suggestion
# with the statically linked debug-wrapper binary that automatically provisions the debugger on a
```

Only worth addressing in case there are other change requests.